kitchen/wazuh-puppet/Dockerfile

```diff
@@ -0,0 +1,79 @@
+ARG DEBIAN_FRONTEND=noninteractive
+
+FROM ubuntu:18.04
+
+# Global required packages
+RUN apt-get update -qq -y && \
+    apt-get upgrade -qq -y && \
+    apt-get install -qq -y \
+      build-essential \
+      apt-transport-https \
+      ca-certificates \
+      curl \
+      software-properties-common \
+      dirmngr \
+      libreadline-dev \
+      libssl-dev \
+      lsb-release \
+      puppet \
+      rbenv \
+      ruby \
+      ruby-dev \
+      rubygems \
+      sudo \
+      systemd \
+      systemd-sysv \
+      wget \
+      zlib1g-dev
+
+# docker-ce related installation tasks.
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add - && \
+    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable" && \
+    apt update -qq -y && \
+    apt-cache policy docker-ce && \
```

Comment:
This lines is not needed, just displays available docker versions.

Reply:
Hi @xr09!

I only use it for debugging purposes.

Kr,

Rshad